cmd/helios-cli/cli_smoke_test.go

```diff
@@ -12,7 +12,6 @@
```

Comment:
Nice E2E. Consider asserting a stable prefix (e.g. `strings.HasPrefix(out, "helios ")`) instead of just non-empty, and make this a table-driven subtest to cover all three entry points: `--version`, `-v`, and `version`.